### PR TITLE
Guides folder and README.md

### PR DESCRIPTION
**Pull request by trademark-living-his-best-life**
Guides page for coding tutorials

### CODE REVIEW

**Comment by FloofyPlasma** [comment]
Hello, thank you for your pull request. I will now be reviewing your contributions. I will let you know if there are any changes required.

File: code-blocks/miscellaneous/run/for-loop.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+description: Clones an object.
```
Comment (FloofyPlasma) [inline code-blocks/miscellaneous/run/for-loop.md:2]: Make sure you update the description too.

**Review by FloofyPlasma — CHANGES_REQUESTED** [review]
I have left comments on areas you should change.

**Comment by FloofyPlasma** [comment]
I am going to convert your PR to a draft, once its done you may change it back.

**Comment by trademark-living-his-best-life** [comment]
> Hello, thank you for your pull request. I will now be reviewing your contributions. I will let you know if there are any changes required.

alright, still gonna work on stuff tommorow

File: guides/simple/for-loops.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  outline:
+    visible: true
+  pagination:
+    visible: true
+---
+*Also refer to code block pages when they come out*
+
+# For Loops
+
+This will not only cover the For Loop block, also but the Loop Through Table, Loop Through Children, and Loop Through Decendants.
```
[the remark this answers is not in the document]
Comment (trademark-living-his-best-life) [reply guides/simple/for-loops.md]: Changed the entire page to just be about Loops.

**Review by FloofyPlasma — APPROVED** [review]

**Pull request merged**